src/main/java/util/DigestMessageGenerate.java

```diff
@@ -6,11 +6,26 @@
 import java.nio.charset.StandardCharsets;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Arrays;
 
 public class DigestMessageGenerate {
 
     private static final Logger logger = Logger.getLogger(BasketDaoImpl.class);
 
+    public static String encryptSha256AndSalt(String password, String salt) {
+        MessageDigest digest = null;
+        try {
+            digest = MessageDigest.getInstance("SHA-256");
+            digest.update(salt.getBytes(StandardCharsets.UTF_8));
+            byte[] encodedHash = digest.digest(password.getBytes(StandardCharsets.UTF_8));
+            return bytesToHex(encodedHash);
+        } catch (NoSuchAlgorithmException e) {
+            logger.error("Message Digest don't have an instance.", e);
+        }
+        return "";
+    }
+
     public static String encryptSha256(String password) {
```

Comment:
does this method do the same that previous? and we use it only in test?

Reply:
what about this comment?

Reply:
It's only for test , in project i use  only encryptSha256AndSalt(String password, String salt)

Reply:
it isn't very good practice. if we use this method only for testing, we should move this method in the test class

Reply:
Ok. i did it